tests/functional/spec/features/hub_and_spoke/hub_and_spoke.spec.js

```diff
@@ -20,6 +22,10 @@ describe('Feature: Hub and Spoke', function () {
     expect($(HubPage.summaryRowState(2)).getText()).to.contain('Not started');
   });
 
+  it('When a user views the Hub, any section with show_on_hub set to false should not appear', function() {
+    browser.openQuestionnaire(hub_and_spoke_schema);
+    expect($(HubPage.submit()).getText()).not.to.contain('Relationships');
```

Comment:
I don't think this is valid. The submit button text would never contain `Relationships`. You should assert that a row for the section that has `show_on_hub` set to `false` is not being displayed on the hub.